src/bundle/ui-dev/src/modules/universal-discovery/components/content-create-widget/content.create.widget.js

```diff
@@ -154,44 +191,22 @@ const ContentCreateWidget = () => {
                         </div>
                         <div className="ibexa-instant-filter__desc">{filtersDescLabel}</div>
                         <div className="ibexa-instant-filter__items">
-                            {contentTypesWithSuggestions.map(([groupName, groupItems], index) => {
+                            {contentTypesDataToShow.map(([groupName, groupItems], index) => {
```

Comment:
just `contentTypesToShow`? Not sure if `Data` tells anything here

Reply:
I like this suggestion :)